.storybook/theme.js

```diff
@@ -0,0 +1,9 @@
+import { create } from '@storybook/theming';
+import Logo from './Logo.png';
```

Comment:
Should this not be in the img folder but not copied over to the build folder.

Reply:
Fixed